build/rewrite.go

```diff
@@ -23,7 +23,7 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/bazelbuild/buildtools/tables"
```

Comment:
please revert this change

Reply:
Done. Sorry about that -- I'm not used to github.